geps/overview.md

```diff
@@ -158,6 +186,16 @@ What is out of scope: see [text from KEP][kep-when-to-use]. Examples:
     practical tests and experiments which are intended to help us learn and
     iterate on the GEP. These can include distributing content, but not under
     any release channel.
+* Q: Should I implement support for `Experimental` channel features?
+  * A: Ultimately one of the main ways to get something into `Standard` is for
+    it to mature through the `Experimental` phase, so we really _need_ people
+    to implement these features and provide feedback in order to have progress.
+    That said, the graduation of a feature past `Experimental` is not a forgone
+    conclusion: **you MUST not implement an experimental feature unless your
+    implementation has a good way to indicate the potential transcience of that
+    feature to its users. You MUST also have a "spin down" plan in place in the
+    event that the GEP never moves forward to Standard and instead ends up being
+    Rejected** (e.g. place the feature behind a feature-gate system).
```

Comment:
Although I agree with what you're saying here, it might be helpful to tweak the language to be a bit less strong:
```suggestion
    That said, the graduation of a feature past `Experimental` is not a forgone
    conclusion. Before implementing an experimental feature, you should:

    * Clearly document that support for the feature is experimental and may disappear in the future
    * Have a plan in place for how you would handle the removal of this feature from the API.
```